Index: qimageblitz-0.0.4/blitz/CMakeLists.txt
===================================================================
--- qimageblitz-0.0.4.orig/blitz/CMakeLists.txt	2007-11-07 12:58:44.000000000 +0100
+++ qimageblitz-0.0.4/blitz/CMakeLists.txt	2008-01-22 01:16:22.000000000 +0100
@@ -17,13 +17,13 @@
 
 CHECK_INCLUDE_FILE(stdint.h HAVE_STDINT_H)
 
-if(NOT MSVC)
-  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"pxor %mm0, %mm0\") ; }" HAVE_MMX)
-  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"xorps %xmm0, %xmm0\"); }" HAVE_SSE)
-  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"xorpd %xmm0, %xmm0\"); }" HAVE_SSE2)
-  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"pi2fd %mm0, %mm0\"); }" HAVE_3DNOW)
-  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"mtspr 256, %0; vand %%v0, %%v0, %%v0\" : : \"r\"(-1) ); }" HAVE_ALTIVEC)
-endif(NOT MSVC)
+#if(NOT MSVC)
+#  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"pxor %mm0, %mm0\") ; }" HAVE_MMX)
+#  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"xorps %xmm0, %xmm0\"); }" HAVE_SSE)
+#  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"xorpd %xmm0, %xmm0\"); }" HAVE_SSE2)
+#  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"pi2fd %mm0, %mm0\"); }" HAVE_3DNOW)
+#  CHECK_CXX_SOURCE_COMPILES(" int main() { __asm__(\"mtspr 256, %0; vand %%v0, %%v0, %%v0\" : : \"r\"(-1) ); }" HAVE_ALTIVEC)
+#endif(NOT MSVC)
 
 SET(BLITZ_LIB_MAJOR_VERSION "4")
 SET(BLITZ_LIB_MINOR_VERSION "0")
